applications/fax/src/fax_maintenance.erl

```diff
@@ -338,3 +341,77 @@ update_job(JobID, State, JObj) ->
                               ),
             'ok'
     end.
+
+-spec migrate_outbound_faxes() -> 'ok'.
+migrate_outbound_faxes() ->
+    migrate_outbound_faxes(?DEFAULT_BATCH_SIZE).
+
+-spec migrate_outbound_faxes(ne_binary() | integer() | wh_proplist()) -> 'ok'.
+migrate_outbound_faxes(Number) when is_binary(Number) ->
+    migrate_outbound_faxes(wh_util:to_integer(Number));
+migrate_outbound_faxes(Number) when is_integer(Number) ->
+    io:format("start migrating outbound faxes with batch size ~p~n", [Number]),
+    migrate_outbound_faxes([{'limit', Number}]);
+migrate_outbound_faxes(Options) ->
+    case couch_mgr:all_docs(?WH_FAXES_DB, Options) of
+        {'error', _E} ->
+            io:format("failed to crawl faxes db: ~p~n", [ _E]);
+        {'ok', []} ->
+            io:format("finished crawling faxes db~n");
+        {'ok', Docs} ->
+            case migrate_outbound_faxes(Docs, []) of
+                [] -> io:format("finished crawling faxes db~n");
+                Result -> migrate_outbound_faxes([{'startkey', next_key(lists:last(Result))}
+                                                  ,{'limit', props:get_value('limit', Options)}
+                                                 ])
+            end
+    end.
+
+-spec migrate_outbound_faxes(wh_json:objects(), list()) -> list().
+migrate_outbound_faxes([], Acc) -> lists:reverse(Acc);
+migrate_outbound_faxes([JObj | JObjs], Acc) ->
+    DocId = wh_doc:id(JObj),
+    maybe_migrate_outbound_fax(DocId),
+    migrate_outbound_faxes(JObjs, [DocId | Acc]).
+
+-spec maybe_migrate_outbound_fax(ne_binary()) -> 'ok'.
+maybe_migrate_outbound_fax(<<"_design/", _/binary>>) -> 'ok';
+maybe_migrate_outbound_fax(DocId) ->
+    case couch_mgr:open_doc(?WH_FAXES_DB, DocId) of
+        {'ok', Doc} -> maybe_migrate_outbound_fax(wh_doc:type(Doc), Doc);
+        {'error', _E} -> io:format("error opening document ~s in faxes db~n", [DocId])
+    end.
+
+-spec maybe_migrate_outbound_fax(api_binary(), wh_json:object()) -> 'ok'.
+maybe_migrate_outbound_fax(<<"fax">>, JObj) ->
+    case wh_json:get_value(<<"pvt_job_status">>, JObj) of
+        <<"failed">> -> migrate_outbound_fax(JObj);
+        <<"completed">> -> migrate_outbound_fax(JObj);
+        _ -> 'ok'
+    end;
+maybe_migrate_outbound_fax(_Type, _JObj) -> 'ok'.
+
+-spec migrate_outbound_fax(wh_json:object()) -> 'ok'.
+migrate_outbound_fax(JObj) ->
+    FromId = wh_doc:id(JObj),
+    {Year, Month, _D} = wh_util:to_date(wh_doc:created(JObj)),
+    FromDB = wh_doc:account_db(JObj),
+    AccountId = wh_doc:account_id(JObj),
+    AccountMODb = kazoo_modb:get_modb(AccountId, Year, Month),
+    kazoo_modb:create(AccountMODb),
+    ToDB = wh_util:format_account_modb(AccountMODb, 'encoded'),
+    ToId = <<(wh_util:to_binary(Year))/binary
+              ,(wh_util:pad_month(Month))/binary
+              ,"-"
+              ,FromId/binary
+            >>,
```

Comment:
You can create these binaries using the `?MATCH_MODB_…` suite. See kazoo_modb_util.erl for example.